docs/source/scheduler-overview.rst

```diff
@@ -105,7 +105,7 @@ The dask collections each have a default scheduler:
 For most cases, the default settings are good choices. However, sometimes you
 may want to use a different scheduler. There are two ways to do this.
 
-1. Using the ``get`` keyword in the ``compute`` method:
```

Comment:
Looks like the `get` kwarg will now raise an exception.
https://github.com/dask/dask/blob/20a6af082856e019a9ae7331ee895544a0da56fd/dask/base.py#L1084-L1085